runtime/moonbase/src/precompiles.rs

```diff
@@ -115,6 +115,17 @@ where
 	R: pallet_evm::Config,
 {
 	fn execute(&self, handle: &mut impl PrecompileHandle) -> Option<PrecompileResult> {
+		// Filter known precompile addresses except Ethereum officials
+		if self.is_precompile(handle.code_address())
+			&& handle.code_address() > hash(9)
+			&& handle.code_address() != handle.context().address
+		{
+			return Some(Err(PrecompileFailure::Revert {
```

Comment:
You can use `revert` from precompile-utils

use precompile_utils::revert;

if self.is_precompile(handle.code_address())
			&& handle.code_address() > hash(9)
			&& handle.code_address() != handle.context().address
		{
		 return Some(Err(revert("cannot be called with DELEGATECALL or CALLCODE")))
		 }